src/Timeline.ts

```diff
@@ -495,12 +495,13 @@ export default abstract class Timeline extends Component {
             event.drawInfo.endText = data.endText;
             if (data.endDate) {
                 const endDate :Date = parseDate(
-                    data.endDate === 'now'
+                    data.endDate === 'ongoing'
```

Comment:
Oh, I already implemented this feature.

Reply:
Yes! I just needed a property to reflect that so it's possible to style the EventAxis accordingly.